### ISSUE TITLE
BUG: google_maps_flutter_web - Polylines not showing up

### ISSUE THREAD

**Issue by real1900** [flutter/flutter#65152]
It seems POLYLINES are not supported. I can see it on Mobile but not on the web. 

Here is what my code looks like: 

```dart
  GoogleMap(
        gestureRecognizers: Set()
          ..add(Factory<PanGestureRecognizer>(() => PanGestureRecognizer()))
          ..add(Factory<ScaleGestureRecognizer>(() => ScaleGestureRecognizer()))
          ..add(Factory<TapGestureRecognizer>(() => TapGestureRecognizer()))
          ..add(Factory<VerticalDragGestureRecognizer>(
              () => VerticalDragGestureRecognizer())),
        tiltGesturesEnabled: true,
        myLocationEnabled: true,
        myLocationButtonEnabled: true,
        mapToolbarEnabled: true,
        buildingsEnabled: true,
        markers: _markers,
        polylines: [
          Polyline(
            width: 4,
            polylineId: PolylineId("1"),
            color: userLocation.urgencyColor,
            points: [
              LatLng(userLocation.latitude, userLocation.longitude),
              UserLocation.kaaba,
            ],
          )
        ].toSet(),
        mapType: widget.mapType,
        zoomGesturesEnabled: true,
        scrollGesturesEnabled: true,
        rotateGesturesEnabled: true,
        compassEnabled: true,
        onMapCreated: _onMapCreated,
        initialCameraPosition: CameraPosition(
          target: LatLng(userLocation.latitude, userLocation.longitude),
          zoom: 19.0,
        ),
      ),
    );
  }
```

**Comment by real1900** [comment]
Thanks for drawing my attention to the Readme. 

It says in the readme:

"There's no defaultMarkerWithHue in web. If you need colored pins/markers, you may need to use your own asset images"

Can someone give me an example on how to add a marker and pins between two locations using my own assets ?

**Comment by hello-addweb** [comment]
Use : google_maps_flutter: ^0.5.28+1

Marker Code :

Set<Marker> markers = Set();
 markers.addAll([
      Marker(
          markerId: MarkerId('value'),
          position: LatLng(12.943450, 77.574100),
          infoWindow: InfoWindow(
          title: "Tital",
            anchor: Offset(1.0, 0.0),
            onTap: (){
              launchURL();
            }
        ),),
    ]);

Then in the build you have to add widget like : 

  GoogleMap(
               mapType: MapType.normal,
           
             markers: markers,
                mapToolbarEnabled: true,
                onMapCreated: (GoogleMapController controller) {
                  _controller.complete(controller);
                  markers.lookup(0).infoWindow.onTap();
                },
              ),

**Comment by ditman** [comment]
> Can someone give me an example on how to add a marker and pins between two locations using my own assets ?

@real1900 there's an example of markers with custom assets in the [example app](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter/example/lib/marker_icons.dart).

Please, let us know if that example helps!

**Comment by real1900** [comment]
Sorry for the confusion - 

I meant POLYLINES are not showing up not MARKERS. 

My apologies

**Comment by real1900** [comment]
@darshankawar @ditman i updated the question

**Comment by ditman** [comment]
> I meant POLYLINES are not showing up not MARKERS.

Polylines should be showing up, check [this sample](https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter/example/lib/place_polyline.dart).

What polyline are you trying to plot?

**Comment by real1900** [comment]
I am basically trying to add one straight polyline between two latlngs .

**Comment by real1900** [comment]
I have shared my code in my original question

**Comment by ditman** [comment]
I'll take a look, this should work, but the example seems to be adding polylines after the fact, instead of adding them on the initial payload of the map. I'll make sure that works.

**Comment by real1900** [comment]
Thanks . Please update me on the progress for this @ditman

**Comment by ditman** [comment]
Taking a look right now.

**Comment by ditman** [comment]
I've posted a PR with a fix for this issue (linked right above this message)

**Comment by ditman** [comment]
The fix for this has been published in [google_maps_flutter_web **v0.1.0+3**](https://pub.dev/packages/google_maps_flutter_web/changelog#0103)
